src/utils/ADempiere/valueFormat.js

```diff
@@ -285,17 +205,19 @@ function getDateFormat({
  * @returns {string} ej: 'qwerty asd' | 'zxc 123'
  */
 export function trimPercentage(stringToParsed) {
-  if (!isEmptyValue(stringToParsed) && String(stringToParsed).includes('%')) {
-    let parsedValue = stringToParsed
-    if (parsedValue[0] === '%') {
-      parsedValue = parsedValue.slice(1)
-    }
+  if (isEmptyValue(stringToParsed) || !String(stringToParsed).includes('%')) {
+    return stringToParsed
+  }
 
-    const wordSize = parsedValue.length - 1
-    if (parsedValue[wordSize] === '%') {
-      parsedValue = parsedValue.slice(0, wordSize)
-    }
-    return parsedValue
+  let parsedValue = stringToParsed
+  if (parsedValue[0] === '%') {
+    parsedValue = parsedValue.slice(1)
   }
-  return stringToParsed
+
```

Comment:
Just can see change for source code with the same functionality, please let it without changes